### DISCUSSION TITLE
Does zilliz cloud support http restful server?

### DISCUSSION THREAD

**Discussion by ShravanSunder** [milvus-io/milvus#27489] [Q&A and General discussion]
Does zilliz cloud support http restful server? 
And will the typescript client support rest calls?

**Comment by ShravanSunder** [comment]
looks like there's a blog post:  https://zilliz.com/blog/getting-started-zilliz-rest-api
It doesn't look like there's an open-api spec or is supported by the client

The restful api does not support full creation of collection:  https://milvus.io/api-reference/restful/v2.3.x/About.md

**Reply by xiaofan-luan** [reply]
currently restful API is high level

https://docs.zilliz.com/docs/insert-entities

but we are working on add more low level API.
